module-2/lab-subsetting-and-descriptive-stats/your-code/main.ipynb

```diff
@@ -2065,83 +2065,27 @@
   },
```

Comment:
- Find the mean and the standard deviation of the temperature of each state.Hint: Use functions from Data Manipulation lesson: You have made it very complex, you don't need so many lines of code. Find below a way of doing this stuff
```Python
temp.groupby("State").agg({'Temperature':['mean','std']})
```
- You have missed the comments to the histogram and the boxplot with the interpretation of each. 
- What is the average Salary per title?: Again too complex.....Find below another possible way....
```Python
employee.groupby("Title").agg({'Salary':'mean'}) 
````
- Good to drop the first column of the orders csv file because it's useless. You can also do it inside read_csv function using the usecols param.